src/containers/Masthead/drawer/TopicMenu.tsx

```diff
@@ -90,7 +90,7 @@ const TopicMenu = ({ topic, subject, onClose, topicPath, onCloseMenuPortion, add
         <DrawerListItem role="none" data-list-item>
           <DrawerHeaderLink
             variant="link"
-            aria-current={location.pathname === topic.path ? "page" : undefined}
+            aria-current={location.pathname.replaceAll("/", "") === topic.path.replaceAll("/", "") ? "page" : undefined}
```

Comment:
Dette fikser indikasjon på valgt emne i menyen. Problemet her var at en ekstra "/" ligger på slutten pathen som brukes i NavigationBox, åpen for andre fikser på problemet om noen har!